scripts/global_tas_average_obs.ipynb

```diff
@@ -30,9 +30,11 @@
     "import os\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #6.    os.environ["ESMFMKFILE"] = (
</pre><p>This can be removed now. Latest versions of ESMF have fixed this problem!</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Ouranosinc/xscen/pull/476/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>